src/main/webapp/script.js

```diff
@@ -11,13 +11,19 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+let currentPage = 1;
+const postPerPage = 10;  // eslint-disable-line
+let numberOfPages = 1;
 
 function addThreads() {  // eslint-disable-line
-  const url = '/videos-sentiment';
+  const url =
+      '/videos-sentiment?currentPage={currentPage}&postPerPage={postPerPage}';
```

Comment:
Does this work as is? Or do you need a leading `$` before the `{`?

Reply:
Yes it works as is, I deployed the page it works.